### PR TITLE
#52 added share buttons

### PR DESCRIPTION
**Pull request by Ferbla**
Added social media buttons for when a user is searched.

### CODE REVIEW

**Comment by jenkoian** [comment]
Hey @Ferbla thanks for this I think it's a nice addition. Unfortunately when trying this out locally it didn't seem to work quite right. I've made a gif for you to see what I was experiencing:

social-buttons

**Comment by Ferbla** [comment]
Well that is embarrassing. I can take a look at this tonight, but it looks as though someone else has also made a share button pull request.

**Comment by jenkoian** [comment]
@Ferbla hey no worries, happy to go with this if you manage to look at the issue above, you were first after all 😃

**Comment by Ferbla** [comment]
@jenkoian they should all be appearing now.

File: views/partials/prs.hbs [hunk 1]
```diff
@@ -14,6 +14,23 @@
 
   <div class="row">
     <div class="large-12 columns">
+      <div class="social-head">
+        <h4><strong>Share:</strong></h4>
+        <div class="social">
+          <ul>
+            <li><div class="twitter"><a href="https://twitter.com/intent/tweet?button_hashtag=hacktoberfest&text={{ prs.length }} / 4 {{ prs.statement }}" target="_blank">Twitter</a></div></li>
```
Comment (jenkoian) [inline views/partials/prs.hbs]: Can we tweak the message to something like "I'm on 3/4 pull requests, how about you? #hacktoberfest"?

**Comment by jenkoian** [comment]
Looks really good @Ferbla! One minor comment about the tweet text but other than that good to go 👍

**Comment by Ferbla** [comment]
@jenkoian Awesome, I just made that change.

**Comment by jenkoian** [comment]
Thanks @Ferbla !

**Pull request merged**